### PR TITLE
Makes travis builds container-based (2fastbuilds)

### PR DESCRIPTION
**Pull request by gbasood**
Thanks baystation

### CODE REVIEW

**Comment by gbasood** [comment]
Should be about as optimal as we can get it
The only big slowdown now is apt packages, and that's all on travis now.

pip package caching is a thing as well, not sure that it really matters but hey

File: .travis.yml [hunk 1]
```diff
@@ -11,22 +12,32 @@ env:
...
+    - $HOME/BYOND-${BYOND_MAJOR}.${BYOND_MINOR}
+    - $HOME/.cache/pip
+
+addons:
+  apt:
+    packages:
+      - libc6-i386
+      - libgcc1:i386
+      - libstdc++6:i386
+      - python
```
Comment (PJB3005) [inline .travis.yml]: Isn't python on travis machines by default?

Comment (gbasood) [inline .travis.yml]: According to https://docs.travis-ci.com/user/ci-environment/#Version-control no, but I'll try to push a commit that removes that line

**Comment by gbasood** [comment]
the biggest timesink for builds now occurs after `loading interface/skin.dmf` but hey nothing we can do there

File: .travis.yml (hunk 1 above)
Comment (PJB3005) [inline .travis.yml]: Could've sworn I read somewhere it was, guess not.

**Comment by clusterfack** [comment]
It doesn't really SEEM faster

**Comment by gbasood** [comment]
Yeah it's actually not much faster for us
I could try caching the interface dir and see if that makes it load faster but probably not, I think it's all on BYOND being slow to build the skin

**Pull request merged**